# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/EarlyHintsService.cpp
# Commit: 1177913e1edf
# Full Hash: 1177913e1edf443da73d438f95a334e466b623e2
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-02 21:26:33
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 8: Add list of open channels from OngoingEarlyHints to RedirectToRealChannelArgs r=necko-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161179
# ==============================================================================

diff -r d33ccbbf407d -r 1177913e1edf netwerk/protocol/http/EarlyHintsService.cpp
--- a/netwerk/protocol/http/EarlyHintsService.cpp	Fri Dec 02 09:45:26 2022 +0000
+++ b/netwerk/protocol/http/EarlyHintsService.cpp	Fri Dec 02 09:45:27 2022 +0000
@@ -100,6 +100,11 @@
   }
 }
 
+void EarlyHintsService::RegisterLinksAndGetConnectArgs(
+    nsTArray<EarlyHintConnectArgs>& aOutLinks) {
+  mOngoingEarlyHints->RegisterLinksAndGetConnectArgs(aOutLinks);
+}
+
 void EarlyHintsService::CollectTelemetry(Maybe<uint32_t> aResponseStatus) {
   // EH_NUM_OF_HINTS_PER_PAGE is only collected for the 2xx responses,
   // regardless of the number of received mEarlyHintsCount.